website/client/src/components/challenges/challengeModal.vue

```diff
@@ -618,6 +632,12 @@ export default {
     toggleCategorySelect () {
       this.showCategorySelect = !this.showCategorySelect;
     },
+    enableSubmit () {
```

Comment:
just one thing, could you make this function throttled so that it's only called every 250ms like [here](https://github.com/HabitRPG/habitica/blob/f8bab04a0e66a819f6d7ca5b210d27d991446bf3/website/client/src/components/challenges/sidebar.vue#L199)? 

The throttle function is a lodash module that can imported this way https://github.com/HabitRPG/habitica/blob/f8bab04a0e66a819f6d7ca5b210d27d991446bf3/website/client/src/components/challenges/sidebar.vue#L92

Reply:
Had no idea about throttling. Sure thing!